manifests/nodejs.yml

```diff
@@ -635,6 +635,9 @@ tests/:
       Test_Debugger_Probe_Statuses:
         "*": irrelevant
         express4: v5.32.0
+        express4-typescript: v5.32.0
```

Comment:
I think this snug in due to your `main` branch being out of date with `origin/main`

```suggestion
```

Reply:
Bad merge. Fixed.